### PR TITLE
625 Enhance hello-monai example

### PR DESCRIPTION
**Pull request by yiheng-wang-nv**
This PR is used to enhance the hello-monai example by supporting monai bundle configs.
Related to #625

### CODE REVIEW

File: examples/hello-monai/config/bundle_configs/train.json [hunk 1]
```diff
@@ -0,0 +1,303 @@
+{
```
Comment (Nic-Ma) [inline examples/hello-monai/config/bundle_configs/train.json:1]: Please don't put the hard-code config of a bundle here.
Let's tell users how to download the spleen bundle and use the config file of downloaded bundle directly.

Thanks.

**Review by Nic-Ma — DISMISSED** [review]

**Comment by Nic-Ma** [comment]
Hi @chesterxgchen @IsaacYangSLA ,

Could you please help review this PR? It's to use MONAI bundle in NVFlare training.

Thanks.

**Comment by YuanTingHsieh** [comment]
@Nic-Ma @yiheng-wang-nv do you want this to go into NVFlare 2.0?
Or you want to update the hello-monai example in the latest branch?
In that case you need to rebase this on top of "dev" branch.

Thanks

File: examples/hello-monai-bundle/README.md [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+
+
+### 1. Install NVIDIA FLARE
+
+Follow the [Installation](https://nvidia.github.io/NVFlare/installation.html) instructions.
+Install additional requirements:
+
+```
+pip3 install monai==0.9.0
+pip3 install fire
```
[the remark this answers is not in the document]
Comment (yiheng-wang-nv) [reply examples/hello-monai-bundle/README.md:13]: Hi @chesterxgchen , this lib is required by `monai.bundle`

File: examples/hello-monai-bundle/README.md [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+unzip spleen_ct_segmentation_v0.1.0.zip
+```
+
+### 3. Set up your FL workspace
+
+Follow the [Quickstart](https://nvidia.github.io/NVFlare/quickstart.html) instructions to set up your POC ("proof of concept") workspace.
+
+### 4. Run the experiment
+
+Log into the Admin client by entering `admin` for both the username and password.
```
[the remark this answers is not in the document]
Comment (yiheng-wang-nv) [reply examples/hello-monai-bundle/README.md:32]: Hi @chesterxgchen , I think it is correct, for other examples `admin` is also used, like in: https://github.com/NVIDIA/NVFlare/tree/dev/examples/hello-cyclic#3-run-the-experiment

**Comment by yiheng-wang-nv** [comment]
Hi @YuanTingHsieh , it seems this PR has many conflicts with the dev branch, let me close it and submit a new one.

**Pull request closed without merging**